pyproject.toml

```diff
@@ -30,10 +30,15 @@ classifiers = [
 dynamic = ["version", "readme"]
 
 dependencies = [
-    "botocore >=1.35.74, <1.35.89", # NOTE: When updating, always keep `project.optional-dependencies` aligned
     "aiohttp >=3.9.2, <4.0.0",
-    "wrapt >=1.10.10, <2.0.0",
     "aioitertools >=0.5.1, <1.0.0",
+    "botocore >=1.35.74, <1.35.89", # NOTE: When updating, always keep `project.optional-dependencies` aligned
+    "python-dateutil >=2.1, <3.0.0",
+    "jmespath >=0.7.1, <2.0.0",
```

Comment:
why are these an urllib3 now required?

Reply:
Again, we use those directly in our own code. It's best practice to declare all dependencies explicitly AFAIK.

I've taken the dependency specifications from upstream to avoid any issues.